proposed/local-sdk-global-json.md

```diff
@@ -0,0 +1,215 @@
+}
+```
+
+These properties will be considered by the resolver host during .NET SDK
+resolution. The `paths` property lists the locations that the resolver should
+consider when attempting to locate a compatible .NET SDK. The `errorMessage`
+property controls what the resolver displays when it cannot find a compatible
+.NET SDK.
+
+This particular configuration would cause the local directory `.dotnet` to be
+considered _in addition_ to the current set of locations. Further if resolution
+failed the resolver would display the contents of `errorMessage` instead of
+the default error message.
+
+## Motivation
+
+There is currently a disconnect between the ways the .NET SDK is deployed in
+practice and what the host resolver can discover when searching for compatible
+SDKs. By default the host resolver is only going to search for SDKs in machine
+wide locations: `C:\Program Files\dotnet`, `%PATH%`, etc ...  The .NET SDK
+though is commonly deployed to local locations: `%LocalAppData%\Microsoft\dotnet`,
+`$HOME/.dotnet`. Many repos embrace this and restore the correct .NET for their
+builds into a local `.dotnet` directory.
+
+The behavior of the host resolver is incompatible with local based deployments.
+It will not find these deployments without additional environment variable
+configuration and instead search in machine wide locations. That means tools
+like Visual Studio and VS Code simply do not work with local deployment by
+default. Developers must take additional steps like manipulating `%PATH%` before
+launching these editors. That reduces the usefulness of items like the quick
+launch bar, short cuts, etc.
+
+This is further complicated when developers mix local and machine wide
+installations. The host resolver will find the first `dotnet` according to its
+lookup rules and search only there for a compatible SDK. Once developers
+manipulate `%PATH%` to prefer local SDKS the resolver will stop considering
+machine wide SDKS. That can lead to situations where there is machine wide SDK
+that works for a given global.json but the host resolver will not consider it
+because the developer setup `%PATH%` to consider a locally installed SDK. That
+can be very frustrating for end users.
+
+This disconnect between the resolver and deployment has lead to customers
+introducing a number of creative work arounds:
+
+- [scripts][example-scripts-razor] to launch VS Code while considering locally
+deployed .NET SDKs
+- [docs and scripts][example-scripts-build] to setup the environment and launch
+VS so it can find the deployed .NET SDKs.
+- [scripts][example-scripts-dotnet] that wrap `dotnet` to find the  _correct_
+`dotnet` to use during build.
+
+These scripts are not one offs, they are increasingly common items in repos in
+`github.com/dotnet` to attempt to fix the disconnect. Even so many of these
+solutions are incomplete because they themselves only consider local deployment.
+They dont't fully support the full set of ways the SDK can be deployed.
```

Comment:
```suggestion
They don't fully support the full set of ways the SDK can be deployed.
```

Reply:
Dont't understand this feedback ... i'm so bad at spelling